File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    char space = ' ';
    char pound = '#';
    int height;

    // Prompt user to input height in between 1 and 8.
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    int a = height;

    // Iterate over given height.
    for (int i = 0; i < height; i++)
    {
        // Iterate over each line
        for (int j = 1; j < a; j++)
        {
            printf("%c", space);
        }

        for (int k = height; k >= a; k--)
        {
            printf("%c", pound);
        }

        // Check if new line is needed.
        if (a > 0)
        {
            printf("\n");
        }
        a--;
    }
}

File: pset6- hello, mario less, mario more, cash, credit, readability, dna/mario/more/mario.py
from cs50 import get_int

# Get height input from user
height = get_int("Height: ")

# Make sure the height is less than 8
while (0 >= height or height > 8):
    height = get_int("Height: ")

# Print double half-pyramid
for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i + "  " + "#" * i)